guides/hack/11-built-in-types/16-string.md

```diff
@@ -110,3 +110,9 @@ HHVM currently supports UTF-8, and single-byte encodings that are supported by t
 Other encodings may be supported in the future, however `Locale\set_native()` is likely to be restricted to
 the current locales; for example, UTF-16 can not be supported by `Locale\set_native()`, as UTF-16 strings can
 contain null bytes.
+
+# Working with `Regex\`
+Functions in the `Regex\` namespace operate on bytes (ASCII strings). If the string being inspected is UTF-8, use a pattern with the `u` flag. Failing to do so may result in one multi-byte character being interpreted as multiple characters. For example:
```

Comment:
Just checking: `Regex\` renders correctly?

Reply:
<img width="1231" alt="image" src="https://user-images.githubusercontent.com/5179225/158903865-4031698c-195e-47f4-90d3-7bc4dbfbff86.png">

Reply:
Ideally, when the namespace reference feature is live, I'd like this to automatically link to that page, much like other functions link directly to their reference page.